### DISCUSSION TITLE
Can the previews show how the email looks like in Mobile ?

### DISCUSSION THREAD

**Discussion by rahulravindran0108** [resend/react-email#589] [Ideas]
### Goals

Consistent interface for folks moving away from Sendgrid design builder

### Background

The issue right now is there is inconsistency when I use Tailwind when it comes to being able to see the email in Desktop vs Mobile. I'm having to send this to my email and specifically use mobile gmail reader to figure out inconsistencies. Is there a better way to test this  right now ?

### Proposal

Introduce a mobile tab in the preview section, so the new interface has the following menu options:
- Desktop
- Mobile
- Source

**Comment by bukinoshita** [comment]
Yup! Should do that.

**Comment by gabrielmfern** [comment]
This has since been done
